src/main/java/com/cleanroommc/groovyscript/compat/mods/ModSupport.java

```diff
@@ -124,6 +125,7 @@ public class ModSupport {
     public static final GroovyContainer<Mekanism> MEKANISM = new InternalModContainer<>("mekanism", "Mekanism", Mekanism::new);
     public static final GroovyContainer<MysticalAgriculture> MYSTICAL_AGRICULTURE = new InternalModContainer<>("mysticalagriculture", "Mystical Agriculture", MysticalAgriculture::new);
     public static final GroovyContainer<LazyAE2> LAZYAE2 = new InternalModContainer<>("threng", "LazyAE2", LazyAE2::new, "lazyae2");
+    public static final GroovyContainer<Lightningcraft> LIGHTNINGCRAFT = new InternalModContainer<>("lightningcraft", "Lightningcraft", Lightningcraft::new);
```

Comment:
`Lightningcraft` => `LightningCraft` (preferably change the class name to be the same case too)